### DISCUSSION TITLE
tblastn

### DISCUSSION THREAD

**Discussion by buckler4** [bbuchfink/diamond#445] [Ideas]
Diamond is really impressive in its speed.  We would like to do `tblastn`, and we see the `eBlastTypeTblastn` - but how do we access it?  Is it implemented?

Thanks.

**Comment by bbuchfink** [comment]
A tblastn mode is not implemented at this time. It is planned but I can't give you a timeline at the moment

**Reply by buckler4** [reply]
Thanks very much.  Is this something that would take lots of coding deep in your codebase, or could this be something that an outside group could assist with?

**Reply by bbuchfink** [reply]
It does not require any complex logic, but the fact that each target sequence consists of 6 translated reading frames that need to be processed together has to be accounted for in a lot of places in the code. It's not an easy modification, but I wouldn't rule it out that an external developer could do that, and I would be willing to assist in this case.
